pkg/daemon/discover/discover.go

```diff
@@ -153,6 +153,12 @@ func rawUdevBlockMonitor(c chan string, matches, exclusions []string) {
 
 	// stdbuf -oL performs line buffered output
 	cmd := exec.Command("stdbuf", "-oL", "udevadm", "monitor", "-u", "-k", "-s", "block")
+	defer func() {
```

Comment:
We already have `	defer stdout.Close() ` which will close this 
So why add an extra log statement??

Reply:
> We already have ` defer stdout.Close() ` which will close this So why add an extra log statement??

sorry, I didn't find stdout.Close() in new version. the old version is the stdout.close was commentted. i will close it